client/src/components/OpeningTimes.jsx

```diff
@@ -7,57 +7,77 @@ const OpeningTimes = ({ currentStation }) => {
 
   return (
     <>
-      <h2>{isOpen ? 'Die Tankstelle ist geöffnet.' : 'Die Tankstelle hat leider gesschlossen.'}</h2>
-      <ul>
+      <OpenStatus>
+        {isOpen ? 'Die Tankstelle ist geöffnet.' : 'Die Tankstelle hat leider gesschlossen.'}
+      </OpenStatus>
+
+      <Week>
         <Day>
           Montag:
-          <time>
+          <p>
```

Comment:
What is bad about the time element? I do not see any advantage when using `p` here instead.